odoo/modules/registry.py

```diff
@@ -964,6 +964,7 @@ def leave_test_mode(self):
     def cursor(self, /, readonly=False):
         """ Return a new cursor for the database. The cursor itself may be used
             as a context manager to commit/rollback and close automatically.
+            If a readonly cursor cannot be obtained, a read-write cursor is returned.
```

Comment:
Let's use the opportunity to document that `cursor(readonly=True)` doesn't matter when there is no replica.
```suggestion
          
            :param readonly: Attempt to acquire a cursor on a replica database. Acquire a read/write cursor on the primary database in case no replica exists or that no readonly cursor could be acquired.
```